File: documentation-examples-java/src/java/dev/typr/foundations/docs/sqlserver/UnicodeStringTypes.java
package dev.typr.foundations.docs.sqlserver;

import dev.typr.foundations.SqlServerType;
import dev.typr.foundations.SqlServerTypes;

@SuppressWarnings("unused")
public class UnicodeStringTypes {
  // start
  SqlServerType<String> ncharType = SqlServerTypes.ncharOf(10);
  SqlServerType<String> nvarcharType = SqlServerTypes.nvarcharOf(255);
  SqlServerType<String> nvarcharMax = SqlServerTypes.nvarcharMax;
  // stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/sqlserver/UnicodeStringTypes.scala
package dev.typr.foundationssc.docs.sqlserver
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object UnicodeStringTypes:
  // start
  val ncharType: SqlServerType[String] = SqlServerTypes.ncharOf(10)
  val nvarcharType: SqlServerType[String] = SqlServerTypes.nvarcharOf(255)
  val nvarcharMax: SqlServerType[String] = SqlServerTypes.nvarcharMax
  // stop
